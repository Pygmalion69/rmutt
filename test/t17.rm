// scope qualifiers: non-local
tests:
 local.top "\n"
 global.top "\n"
 non_local.top "\n"
;

package local;

top: A X;

A: (X="1") B X;
B: (X="2") C X;
C: (X="3") X;

package non_local;

top: A X;

A: (X="1") B X;
B: (X="2") C X;
C: (^X="3") X;

package global;

top: A X;

A: (X="1") B X;
B: (X="2") C X;
C: ($X="3") X;
