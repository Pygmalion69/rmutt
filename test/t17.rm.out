321local.X
2213
331non_local.X

